CHANGELOG.md

```diff
@@ -204,6 +204,7 @@ All notable changes for each version of this project will be documented in this
 
 - `IgxCombo`:
     - Added `autoFocusSearch` input that allows to manipulate the combo's opening behavior. When the property is `true` (by default), the combo's search input is focused on open. When set to `false`, the focus goes to the combo items container, which can be used to prevent the software keyboard from activating on mobile devices when opening the combo.
+    - Make `onSearchInput` event cancellable. The event args type has been changed to `IComboSearchInputEventArgs`, which have two properties: `change` - holds the change in the search input and `cancel` - indicates whether the event should be canceled.
```

Comment:
Please rename "change" to "value" or similar, which will not lead to the understanding that arguments hold the difference from the previous time the event was fired.

Reply:
Renamed to 'searchTerm'